src/main/java/edu/cornell/kfs/concur/service/impl/ConcurFailedEventQueueProcessingController.java

```diff
@@ -0,0 +1,39 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+import org.kuali.kfs.sys.KFSConstants;
+
+import edu.cornell.kfs.rass.RassConstants;
+
+public enum ConcurFailedEventQueueProcessingController {
+    OFF(false, false, false),
+    READONLY(true, true, false),
+    READWRITE(true, false, true);
+    
+    public final boolean queryFailedEventQueue;
+    public final boolean logFailedEventQueue;
+    public final boolean writeFailedEventQueue;
```

Comment:
The name of the enum class already makes it clear that it's affecting the capabilities of the failed-event queue, so it might be helpful to rename these flags to something less redundant.

Reply:
how about
queryQueue
logQueue
persistQueue

Reply:
Yes, those replacement names sound good. Thanks!